t/43-cli-clone.t

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Test::Warnings qw(:report_warnings warning);
+use Mojo::Base -strict;
+
+use FindBin;
+use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
+
+use Capture::Tiny qw(capture capture_stdout);
+use Mojo::Server::Daemon;
+use Mojo::File qw(tempdir tempfile);
+use OpenQA::CLI;
+use OpenQA::CLI::clone;
+use OpenQA::Test::Case;
+use OpenQA::Test::TimeLimit '10';
+
+OpenQA::Test::Case->new->init_data(fixtures_glob => '01-jobs.pl 03-users.pl 05-job_modules.pl');
```

Comment:
do you actually need the job modules? The list of fixtures should be at best empty, e.g. as in t/10-jobs-results.t to deliver fastest test results. Only if necessary fixtures should be added but only the actually necessary fixtures then.